istioctl/pkg/root/root.go

```diff
@@ -49,5 +49,6 @@ func defaultLogOptions() *log.Options {
 	o.SetOutputLevel("klog", log.WarnLevel)
 	o.SetOutputLevel("kube", log.ErrorLevel)
 	o.SetOutputLevel("controllers", log.WarnLevel)
+	o.SetOutputLevel("util", log.WarnLevel)
```

Comment:
should we just change everything to warn, and then opt-in others that we want at different levels

Reply:
That would be easy to maintain

Reply:
Makes sense